gfx.cpp

```diff
@@ -4015,19 +4015,10 @@ class GfxInternal
             return GFX_SET_ERROR(kGfxResult_InvalidParameter, "Cannot dispatch using an invalid sbt object");
         if(!width || !height || !depth)
             return kGfxResult_NoError;  // nothing to dispatch
-        uint32_t const max_num_groups = 65535; // AMD doesn't allow to dispatch more than 65535 groups at once
-        if(width > max_num_groups || height > max_num_groups || depth > max_num_groups)
+        uint32_t const max_size = 2UL << 30;
+        if(width * height * depth > max_size)
```

Comment:
The only limitation DispatchRays has is that the total size cannot be greater than 2^30 (as per the spec)